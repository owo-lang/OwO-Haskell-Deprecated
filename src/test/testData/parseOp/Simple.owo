module SimpleOp where
  infixl 1 +
  f = 1 + 1
