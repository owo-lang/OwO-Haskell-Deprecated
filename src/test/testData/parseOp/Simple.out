File type: owo
Top module name: .SimpleOp
 implementation of function  f (37 3 3) (38 3 4)
  no pragmas
  clause for  f (37 3 3) (38 3 4)
  patterns matching parsed as expression
   named reference f (37 3 3) (38 3 4)
  function body
   function application
   function being applied
    function application
    function being applied
     named reference + (43 3 9) (44 3 10)
    value applied to the function
     constant (41 3 7) (42 3 8)
      IntegerLit 1
   value applied to the function
    constant (45 3 11) (46 3 12)
     IntegerLit 1
  no with abstraction
  no where clause
